tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -572,6 +572,13 @@ considered a comment.
     The list of highlighted keywords and their colors can be changed in the **Text
     Editor > Theme > Comment Markers** section of the Editor Settings.
 
+It is also possible to mark certain comments as
+:ref:`documentation comments <doc_gdscript_documentation_comments>` by using
+two hash symbols (``##``) instead of one (``#``). This only has an effect for
+comments placed immediately *above* a documentable item (such as a member
+variable), or at the top of a file. Dedicated formatting options are also
+available. See :ref:`doc_gdscript_documentation_comments` for details.
+
```

Comment:
```suggestion
Use two hash symbols (``##``) instead of one (``#``) to add a
:ref:`documentation comment <doc_gdscript_documentation_comments>`, which will
appear in the script documentation and in the inspector description of an
exported variable. Documentation comments must be placed directly *above* a
documentable item (such as a member variable), or at the top of a file. 
Dedicated formatting options are also available.
See :ref:`doc_gdscript_documentation_comments` for details.

::

    ## This comment will appear in the script documentation.
    var value
    ## This comment will appear in the inspector tooltip, and in the documentation.
    @export var exported_value

```
(And also move this whole section to *before* the tip about special keywords)

Includes two examples of the most common use cases. Uses more active phrasing.

Linking to the same page twice in a paragraph seems a bit odd, but it's fine.